ical.js

```diff
@@ -138,9 +138,19 @@ function getTimeZone(value) {
   // And offset is still present
   if (tz && tz.startsWith('(')) {
     // Extract just the offset
-    const regex = /[+|-]\d*:\d*/;
+    const offsetcomps = tz.match(/([+|-]\d+):?(\d+)/);
```

Comment:
Please explain why exactly you changed that regex and what the added lines following this change try to fix/change.

Reply:
I try to map indications like `(GMT +02:00)` to matching [IANA etcetera](https://data.iana.org/time-zones/tzdb/etcetera) zones, if available. The advantage of those IANA zone names is that they can be processed by moment-timezone.